index.js

```diff
@@ -64,6 +72,12 @@ function JestJUnit (globalConfig, options) {
   this._globalConfig = globalConfig;
   this._options = options;
 
+  this.onTestResult = (test, testResult, aggregatedResult) => {
+    if (testResult.console && testResult.console.length > 0) {
+      consoleBuffer[testResult.testFilePath] = testResult.console;
+    }
+  };
+
   this.onRunComplete = (contexts, results) => {
     processor(results, this._options, this._globalConfig.rootDir);
```

Comment:
you could read `consoleBuffer` here and add it back to each result instance before you pass it to `processor`.

Reply:
> Interesting, yep! You're right! Thanks so much for the help there. This is a really helpful feature.
Hi,
I tried executing the test from your repository but still not able to see the console logs on JUnit xml. Did it worked when you tried it?